apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/nodes/SocialPreviewNode/SocialPreviewNode.tsx

```diff
@@ -101,32 +109,18 @@ export function SocialPreviewNode(): ReactElement {
             height="30px"
             justifyContent="space-between"
             alignItems="center"
+            spacing={1}
           >
-            <Avatar
-              sx={{
-                height: 15,
-                width: 15,
-                bgcolor: 'background.default',
-                color: 'background.paper',
-                mr: 1.5
-              }}
-            >
-              <UserProfile2Icon sx={{ height: '15px' }} />
-            </Avatar>
+            <UserProfileCircle sx={{ fontSize: 14, color: bgcolor }} />
             <Box flexGrow={1}>
               <Box
                 width={45}
                 height={9}
-                bgcolor="background.default"
+                bgcolor={bgcolor}
                 borderRadius="4.5px"
               />
             </Box>
-            <Box
-              width={9}
-              height={9}
-              bgcolor="background.default"
-              borderRadius="4.5px"
-            />
+            <Box width={9} height={9} bgcolor={bgcolor} borderRadius="4.5px" />
```

Comment:
Use theme token for this
```suggestion
            <Box width={9} height={9} bgcolor={bgcolor} borderRadius={1} />
```